MkBuildBot/Program.cs

```diff
@@ -74,8 +74,10 @@ private async Task SlashCommandHandler(SocketSlashCommand command)
                     var gameVersion = (long) command.Data.Options.First(x => x.Name == "game-version").Value;
                     var excludeInlineParam = (long) command.Data.Options.First(x => x.Name == "exclude-inline-bikes").Value;
                     var amount = (long) (command.Data.Options.FirstOrDefault(x => x.Name == "amount")?.Value ?? (long) 1);
+                    bool isDLCCharactersExcluded = (long)command.Data.Options.First(x => x.Name == "exclude-dlc-characters").Value == 1;
 
-                    var excludeInline = excludeInlineParam != 0; //!= => true | == => false
+
+					var excludeInline = excludeInlineParam != 0; //!= => true | == => false
```

Comment:
Same as above and why are here two blank lines?

As I don't have anything against using the specific type for the variable I would like to use `var` here as well, as it fits the code style here. Not forgetting to mention that the order is wrong. `excludeInline` should be before `isDLCCharactersExcluded` as it should match the order as above, same goes for `amount`

Also not blank spaces here as they all belong together.